### PR TITLE
[#10042] Session Submission Page optimized UI for mobile compatibility

### LINKED ISSUE #10042: Improve UI for Mobile Compatibility
Several pages within TEAMMATES are not optimised for mobile screen widths. Text gets truncated or content goes missing. This epic is aimed at improving the mobile compatibility of UI across the web application through small but many front-end edits.

- Environment: v7 Dev Environment

**First Timers Only**
Only first timers should be submitting PRs for this epic. Additionally, in the spirit of our [contributing guidelines](https://github.com/TEAMMATES/teammates/blob/master/docs/CONTRIBUTING.md#phase-c-start-contributing) we will only be accepting one PR per person addressing this issue. Submitted PRs should target **not more than one component** (i.e. only one html file or set of html/scss/ts  depending on the nature of the fix) along with any associated tests if necessary (e.g. snapshots or changes to spec.ts). An example of such a PR is #10037

**Expectations**
Submitted PRs should minimally include a screenshot for manual verification of the fix. Please refer to our [development guideline](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md) on how to work with your copy of TEAMMATES to achieve this. PRs addressing this epic should not "Resolve" the issue but rather be "Part of" the issue (i.e. do not use [GitHub auto-closing keywords](https://help.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords) with your PR). An example of this is #10052 

Please also try to run the component tests locally. If the file you're working on has a snapshot, you will need to update that snapshot to pass the CI check. You can refer to the testing guide [here](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#testing) and more information about snapshot testing [here](https://github.com/TEAMMATES/teammates/blob/master/docs/snapshot-testing.md). You can refer to https://github.com/TEAMMATES/teammates/pull/9997#issuecomment-605014375 for a more involved guide if you're still encountering problems, or just put up a draft PR and ask for help :)

**Working List**
This list tracks the pages or components we are aware of that need mobile UI optimisation. It is not a comprehensive list, and if you find something not on this list, feel free to submit a PR or comment here in this issue for feedback! :)

- [x] Instructor Pages: [Edit Feedback Session Page](https://user-images.githubusercontent.com/42177597/81633942-6fa31d00-9440-11ea-8715-f568884e1a52.gif)
- [x] Instructor Pages: Add Feedback Session Page
- [x] Instructor Pages: Course Details & Edit Pages
- [x] Instructor Pages: Course Enrol Students Page
- [x] Student Pages: Feedback Session Result Page
- [x] Session Submission Page
- [x] MCQ Single Answer Question
- [x] Rank Options Question
- [x] Webpage Bottom Banner

### PR DESCRIPTION
**Pull request by edwardkang0925**
Part of #10042 

**Outline of Solution**
Modified bootstrap styling by using the grid system. 

**Concern**
When we run the 'npm run lint', it complains about "File: /Users/edwardkang/dev/teammates/src/main/webapp/ngsw.json
Line: 360 Unexpected additional newlines at the end of the file. [warning]" but we haven't changed anything. Please let us know if it is a problem with the code we have modified.

**Original** 
initial
[](url)
The words are cracked when the screen width is narrow. 

**After Change**
When the screen is narrow, 
after_fix_narrow

When the screen is wide, 
after_fix_wide

### CODE REVIEW

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.scss [hunk 1]
```diff
@@ -4,3 +4,23 @@
...
+
+.col-sm-3 {
+  text-align: center;
+}
+
+.col-sm-6 {
+  text-align: center;
+}
+
+$breakpoint-tablet: 577px;
```
Comment (madanalogy) [inline src/web/app/pages-session/session-submission-page/session-submission-page.component.scss:16]: Is this really necessary?

**Comment by madanalogy** [comment]
> Fixes #10042

Please read the issue details, you shouldn't be including auto-closing keywords in your PR.

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.scss (hunk 1 above)
Comment (edwardkang0925) [reply src/web/app/pages-session/session-submission-page/session-submission-page.component.scss:16]: Hello! Thank you so much for reviewing our pull request. 
After thinking again, this portion of code can be removed, and we can write "min-width: 768px" (after changing sm to md) at the media query below and add "text-center" for each row. 
Am I on the right track? 
Thank you again for reviewing our PR!

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.scss [hunk 2]
```diff
@@ -4,3 +4,23 @@
...
+
+@media (min-width: $breakpoint-tablet) {
```
Comment (edwardkang0925) [inline src/web/app/pages-session/session-submission-page/session-submission-page.component.scss:18]: `@media (min-width: 768px) {`

File: src/web/app/pages-session/session-submission-page/session-submission-page.component.scss (hunk 1 above)
Comment (madanalogy) [reply src/web/app/pages-session/session-submission-page/session-submission-page.component.scss:16]: Yeap, looks okay to me

Comment (edwardkang0925) [reply src/web/app/pages-session/session-submission-page/session-submission-page.component.scss]: We have modified our PR. Could you please review our modification? Thank you so much!

**Pull request merged**